### ISSUE TITLE
ListItem has background color of backgroundStrong by default

### ISSUE THREAD

**Issue by sallar** [tamagui/tamagui#941]
### Current Behavior

The `ListItem` component now has a `backgroundColor` property with value of `$backgroundStrong` which was added some time ago:

https://github.com/tamagui/tamagui/commit/d5b9842f063f100f57c52ca71c556d1c2673a8ab#diff-1454dfa1f65c14e091433ef66b30738c489824c35be0a62b04db86726211b52a

I think this is a mistake

### Expected Behavior

The ListItem should not have any background set by default, or if it is, it should be `$background`

### Tamagui Version

```markdown
1.14.4
```

### Reproduction

```markdown
Use `ListItem` component. You will see a backgroundStrong background color.
```

**Comment by natew** [comment]
This is fixed in the upcoming release #883

**Comment by sallar** [comment]
@natew This is not fixed in the new release. It's broken in a different way.

The `ListItem` component now has a strong background. 
If I overwrite them `themes` key in `tamagui.confg.ts` like this:

```
themes: {
  dark: themes.dark,
  light: themes.light
}
```
the issue goes away.
but if I do this:

```
{
  themes
}
``` 
and import the `themes` object from `@tamagui/themes` it breaks again.
I did some investigating and realized the issue comes from `allThemes` that's defined in `@tamagui/themes/src/themes.tsx`. `themes.dark_ListItem` has the value:

```
themes.

{"background": "#050505", "backgroundFocus": "#050505", "backgroundHover": "#151515", "backgroundPress": "#191919", "backgroundStrong": "#050505", "backgroundTransparent": "#050505", "borderColor": "#191919", "borderColorFocus": "#191919", "borderColorHover": "#232323", "borderColorPress": "#151515", "color": "#fff", "colorFocus": "#fff", "colorHover": "#fff", "colorPress": "#fff", "colorTransparent": "#fff", "placeholderColor": "#626262"}
```

as you can see the values of `background`, `backgroundStrong` and `backgroundTransparent` are the same.
